### ISSUE TITLE
ljust() not working with rpy python 3 enabled

### ISSUE THREAD

**Issue by xavi-mat** [renpy/renpy#2551]
The function `ljust()` applied to a string worked just fine until I tried the `rpy python 3` feature.

It throws the exception: 
`TypeError: ljust() argument 2 must be char, not unicode`

(this probably affects too to `str.rjust()` and `str.center()`...)

```
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 373, in script
    pause
  File "renpy/common/000statements.rpy", line 439, in execute_pause
    renpy.pause()
  File "renpy/common/00action_other.rpy", line 537, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/functions.rpy", line 1349, in refreshwatcher
    lin5a = "consc.:" + ("#" * consci11).ljust(11, "·")
TypeError: ljust() argument 2 must be char, not unicode

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 373, in script
    pause
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/ast.py", line 1960, in execute
    self.call("execute")
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/ast.py", line 1948, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/statements.py", line 278, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 439, in execute_pause
    renpy.pause()
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/exports.py", line 1519, in pause
    rv = renpy.ui.interact(mouse='pause', type='pause', roll_forward=roll_forward)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/ui.py", line 298, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/core.py", line 2978, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/core.py", line 3378, in interact_core
    root_widget.visit_all(lambda i : i.per_interact())
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/core.py", line 566, in visit_all
    d.visit_all(callback, seen)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/core.py", line 566, in visit_all
    d.visit_all(callback, seen)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/core.py", line 566, in visit_all
    d.visit_all(callback, seen)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/core.py", line 566, in visit_all
    d.visit_all(callback, seen)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/screen.py", line 432, in visit_all
    callback(self)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/core.py", line 3378, in <lambda>
    root_widget.visit_all(lambda i : i.per_interact())
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/screen.py", line 443, in per_interact
    self.update()
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/screen.py", line 656, in update
    i.set_transform_event(self.current_transform_event)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/behavior.py", line 2240, in set_transform_event
    run(self.action)
  File "/home/xavimat/Documents/renpy-7.4.0-sdk/renpy/display/behavior.py", line 323, in run
    return action(*args, **kwargs)
  File "renpy/common/00action_other.rpy", line 537, in __call__
    rv = self.callable(*self.args, **self.kwargs)
  File "game/functions.rpy", line 1349, in refreshwatcher
    lin5a = "consc.:" + ("#" * consci11).ljust(11, "·")
TypeError: ljust() argument 2 must be char, not unicode
```

**Comment by renpytom** [comment]
This should be fixed by 216e5ed95 .

**Comment by xavi-mat** [comment]
Checked in 7.4.1. It seems solved.
("Force Recompile" is needed).
